.editorconfig

```diff
@@ -28,3 +29,12 @@ indent_size = 4
 
 [Makefile]
 indent_style = tab
+indent_size = 8
```

Comment:
I'm not sure what the general preference on tab width for *.go/Makefile is. 8 seems a archaic to me but it is what vim and GitHub use by default. I personally prefer 2 for everything, but I guess not everyone might agree to that.

Reply:
4 is good compromise but I also prefer 2

Reply:
I will update all of them to 2, let's see if it gets some acceptance. This should especially help with template code where the 8-wide tabs really get in the way when one it already 10+ levels deep in indentation.